Resources/Prototypes/Tiles/plating.yml

```diff
@@ -6,7 +6,7 @@
   isSubfloor: true
   footstepSounds:
     collection: FootstepPlating
-  friction: 0.2
+  friction: 1
```

Comment:
This was last edited by Skye to be 0.2 in her pulling speed fix PRs, are you sure this is correct? It might be something else at play unless friction now works differently

Reply:
Nvm it got changed in https://github.com/space-wizards/space-station-14/pull/36847